modules/oe_subscriptions_anonymous/oe_subscriptions_anonymous.module

```diff
@@ -71,6 +71,13 @@ function oe_subscriptions_anonymous_theme(array $existing, string $type, string
         'attributes' => [],
       ],
     ],
+    'oe_subscriptions_anonymous_subscriptions_link' => [
```

Comment:
We shouldn't add templates for all the links, we did it for the flag because it's a special case. Anyway see below why this is not needed.

Reply:
Ok, removing